app/models/subject.rb

```diff
@@ -11,8 +11,4 @@ def self.sorted_by_number
   def self.with_assessments
     joins(:assessments).distinct
   end
-
-  def to_s
```

Comment:
It is inherited from `Object`, yes. You are meant to override it with a valid string representation of the object. That's often useful in debugging and ocasionally other situations, but we're likely overusing `to_s` in the views.